server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -9,6 +9,9 @@
 import edu.jhu.server.data.CaseFile;
 
 public class Game {
+	private enum EventType {
+		TEST, CHAT, GAMESTART, SUGGESTION, TURNNOTIFICATION
```

Comment:
can you add _REQUEST/_NOTIFICATION to each of the event types? itll be good to be explicit. also this is nitpicky but can you separate words with underscores (e.g. GAMESTART -> GAME_START)